packages/wells/README.md

```diff
@@ -126,33 +152,57 @@ const rootId = 4438800495523058;
 const response = await client.wellbores.listByWellId(rootId, fn);
 ```
 
-### polygon search with additional filters
+#### _Read wellbore trajectories and rows:_
 
-```js
-const polygon =
-  'POLYGON ((-4.86423 63.59999, 19.86423 63.59999, 19.86423 52.59999, -4.86423 52.59999, -4.86423 63.59999))';
+Trajectories are expressed as **Survey[]** and are found on the wellbore object using a lazy getter _wellbore.trajectory()_. All trajectory rows are found on the trajectory object (**Survey** dataclass) and can be acessed using a lazy getter _trajectory.rows(limit=1000)_.
 
-const response = await client.wells.listWells(
-  {
-    wellGeometry: {
-      geometry: polygon,
-      crs: 'epsg:4326',
-      outputCrs: 'EPSG:4326',
-    },
-    filter: {
-      name: ['Well A'],
-      dataSource: ['A', 'B'],
-      operator: ['A', 'B'],
-      field: ['A', 'B'],
-      block: ['A', 'B'],
-    },
-    limit: 1000,
-  }
+```js
+const wellId = 2278618537691581;
+const wellbores = await client.wellbores.listChildren(wellId);
+wellbores.forEach(async wellbore => {
```

Comment:
FYI async forEach is sometimes confusing, do we want to recommend using it?

often when we write code like this we actually meant for a more synchronous for-of type thing

https://advancedweb.hu/how-to-use-async-functions-with-array-foreach-in-javascript/